BrokenText.md

```diff
@@ -0,0 +1,9 @@
+# Unturned- Broken text error
+ If you have joined a server, that has too many placards/signs/flags with a lot of text on them in a specific area you might experience the broken text error. That error will cause some of the text written on the placards/signs/flags to either turn into some weird shapes or even disappear. Sometimes that affects all text including the in-game chat and the main menu.
+
+###Solution
```

Comment:
```suggestion
## Solution
```
Markdown formatting requires a space between the `#` for headers and the following text